### ISSUE TITLE
UVM demo notebook fails to run on centOS

### ISSUE THREAD

**Issue by robocopnixon** [rapidsai/cugraph#2199]
**Describe the bug**
After the data is downloaded and decompressed, the notebook does not proceed further and just hangs forever. 

**Steps/Code to reproduce bug**
Simply run all cells in the container listed below.

**Expected behavior**
The notebook should run, or at least fail with errors. 

**Environment overview (please complete the following information)**
 - Environment location: Docker
 - Method of cuGraph install: Docker
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used

`docker pull rapidsai/rapidsai-core-nightly:22.04-cuda11.2-runtime-centos7-py3.8`
`pmahler@dt02:~$ docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786     -v ~/extra:/rapids/notebooks/extra rapidsai/rapidsai-core-nightly:22.04-cuda11.2-runtime-centos7-py3.8`

**Comment by robocopnixon** [comment]
Actually, during the time it took me to fill out the bug report, the notebook did proceed another step. The output I got was `Reader:  412.2092275619507 s` and it hasn't finished yet. This seems very slow for a demo of speed. Is this expected behavior?

**Comment by robocopnixon** [comment]
I've been sitting here for ten minutes and I'm giving up. This is running on a t4.

**Comment by charlesbluca** [comment]
I'm not really familiar with what speeds we should be expecting for large (>25GB) CSV reads in cuDF, but I can say anecdotally that I manually set the `nrows` parameter down as suggested in the notebook caption and was able to get things working (granted I ended up doing this because WSL2 was giving immediate OOMs during the read, xref https://github.com/rapidsai/cudf/issues/10554).

Perhaps we should make a more visible "switch" to control how much of the dataset we're reading into memory? The caption does suggest using `nrows` but maybe it would be better to add a code block where users can comment/uncomment to enable/disable reading the entire dataset.
